app/components/metatip/metatip.element.css

```diff
@@ -213,8 +216,98 @@
   box-shadow: 0 0 0 1px var(--theme-icon_active-bg);
 }
 
+:host [color-swatches] {
+  display: flex;
+  width: 100%;
+  height: 7em;
+  flex-direction: row;
+  padding: 0;
+  margin: 0 0 5px;
+
+  @media (max-width: 700px) {
+    flex-direction: column;
+    height: 14em;
+  }
+}
+
+:host [color-swatch] {
+  flex: 1;
+  width: 22ch;
+  height: 7em;
+  display: grid;
+  position: relative;
+  align-content: end;
+  box-sizing: border-box;
+  padding: 0.5em;
```

Comment:
```suggestion
  padding: 10px;
```

this aligns the color-swatch content with the accessibility content, was just a hair off when using ems in this spot. screenshot shows the result of switching to pixels 👍🏻 

![Screen Shot 2021-05-13 at 11 45 30 AM](https://user-images.githubusercontent.com/1134620/118171996-ba070300-b3e0-11eb-872d-bada9180c69f.png)